azure-pipelines.yml

```diff
@@ -28,13 +28,12 @@ jobs:
       python -m pip install --upgrade pip
       pip install .
     displayName: 'Create Conda environment for DeepForest'
-  
+
   - script: |
       pip install pytest pytest-azurepipelines yapf
       pytest
```

Comment:
Can you add a v(verbose) to the pytests?
`pytest -v`

Reply:
Okay